src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/General/PreferencesType.php

```diff
@@ -131,6 +132,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'Allow iframes on text fields like product description. We recommend that you leave this option disabled.',
                     'Admin.Shopparameters.Help'
                 ),
+                'attr' => [
+                    'multistore_configuration_key' => 'PS_ALLOW_HTML_IFRAME',
```

Comment:
It shouldn't be an attr like that but you should use data behavior
```suggestion
                    'data-multistore_configuration_key' => 'PS_ALLOW_HTML_IFRAME',
```

Reply:
well this behavior has already been approved: I can't improve this in this PR because it's a generic behavior

Reply:
omg :scream: 
I'm afraid of side effects (Invalid HTML attributes) when parsing it.
It's not compatible with HTML validator:
https://validator.w3.org/nu/#textarea
![image](https://user-images.githubusercontent.com/1462701/120771634-b5d18f80-c51f-11eb-916f-4140a1ca4a0d.png)

Is it already on 1.7.8.x or only on develop?

Reply:
It's on 1.7.8.x and in the dev docs already :D 

https://devdocs.prestashop.com/1.7/development/multistore/configuration-forms/

Do you think it is such a big deal that it must be fixed on 1.7.8.x ?

Reply:
It must be fixed in the 1.7.8.x

Reply:
I think I'm going to change this in another PR, that will need to be merged before this one